editor/plugins/animation_blend_tree_editor_plugin.cpp

```diff
@@ -1262,4 +1264,164 @@ AnimationNodeBlendTreeEditor::AnimationNodeBlendTreeEditor() {
 	open_file->set_title(TTR("Open Animation Node"));
 	open_file->set_file_mode(EditorFileDialog::FILE_MODE_OPEN_FILE);
 	open_file->connect("file_selected", callable_mp(this, &AnimationNodeBlendTreeEditor::_file_opened));
+
+	animation_node_inspector_plugin = Ref(memnew(EditorInspectorPluginAnimationNodeAnimation));
+	EditorInspector::add_inspector_plugin(animation_node_inspector_plugin);
+}
+
+AnimationNodeBlendTreeEditor::~AnimationNodeBlendTreeEditor() {
+	EditorInspector::remove_inspector_plugin(animation_node_inspector_plugin);
+}
```

Comment:
```suggestion
AnimationNodeBlendTreeEditor::~AnimationNodeBlendTreeEditor() {
}
```
This is required for add-on in order to remove it when disabling the add-on, but it is not necessary to built-in plugin.